site/src/components/AuditLogRow/AuditLogRow.tsx

```diff
@@ -31,6 +31,14 @@ const pillTypeByHttpStatus = (
 }
 
 const readableActionMessage = (auditLog: AuditLog) => {
+  // workspace builds audit logs don't have targets; therefore format them differently
+  if (auditLog.resource_type === "workspace_build") {
+    const substr = auditLog.description.trim().split(" ").pop() ?? ""
+    return auditLog.description
+      .replace("{user}", `<strong>${auditLog.user?.username}</strong>`)
+      .replace(substr, `<strong>${substr}</strong>`)
+  }
```

Comment:
Good catch!!!!!! 
I wish we could use `workspace` as the actual target, but I'm not sure how to do that. I think it would involve some surgery on the main audit function. I would like to look into that at some point, though.

I think for now, the best option is the `additional_fields` solution! I'll do that, thanks!